src/data/markdown/docs/03 cloud/05 Project and Team Management/03 Projects.md

```diff
@@ -11,11 +11,11 @@ You can think of Projects as a filing system that organizes your tests on a per-
 
 Here are some ways we users use Projects to stay organized:
 
-- Per team: Each team is given their own project for them to do their work and collaborate.
-- Per component: A project is created for each component or service you're testing.
-- Per brand: E-commerce customers may want to use projects per brand to stay organized.
-- Per Customer: When dealing with custom software, you might organize by customer to ensure each unique system is tested.
-- Per Major Release: After your systems go through a major change, you might create a new project to organize the most recent data.
+- Per team: Each team is given their own project for them to do their work and collaborate (see also [Teams](/cloud/project-and-team-management/teams)).
+- Per component: A project is created for each component or service you're testing
+- Per brand: E-commerce customers may want to use projects per brand to stay organized
+- Per Customer: When dealing with custom software, you may wish to organize by customer to ensure each unique system is tested
+- Per Major Release: After your systems go through a major change, you may wish to create a new project to organize the most recent data
```

Comment:
```suggestion
- Per Customer: When dealing with custom software, you can organize by customer to ensure each unique system is tested
- Per Major Release: After your systems go through a major change, you can create a new project to organize the most recent data
```

Reply:
We use the Google style guide, via inheritance from Google:

> wish
>   Don't use. Instead, use a word like want or need. [^1]

I think this is because "wish" conveys a certain level of emotion and desire that may not be appropriate for enterprise SaaS environments.
[^1]: https://developers.google.com/style/word-list?hl=en#letter-w